raiden/transfer/channel.py

```diff
@@ -128,6 +132,19 @@ def is_lock_locked(
     return secrethash in end_state.secrethashes_to_lockedlocks
 
 
+def is_lock_expired(
+        locked_lock: LockedTransferSignedState,
```

Comment:
I know you're going to ask the question :smile: so i decided to take the lock as a param rather than the channel state because the lock would need to also be fetched from the caller side. So i fetch it there, check for expiry using this function and then use the lock instance in the caller.